swift/ql/lib/codeql/swift/elements/decl/Decl.qll

```diff
@@ -1,5 +1,5 @@
 private import codeql.swift.generated.decl.Decl
 
 class Decl extends Generated::Decl {
-  override string toString() { result = super.toString() }
+  override string toString() { result = this.getPrimaryQlClasses() }
```

Comment:
I don't think we should "fix" the FP like this. Really, this is a FP that should be fixed in the QL-for-QL query, and I don't think we should worsen the quality of our `toString` to make the FP go away (remember that the reason we have a `toString` is to create useful alert messages, and I don't think we should have QL class names in alert messages).

Reply:
So I suggest we revert this commit, and open an issue in the internal QL-for-QL repo.